libethereum/State.cpp

```diff
@@ -21,720 +21,792 @@
 
 #include "State.h"
 
-#include <boost/filesystem.hpp>
-#include <boost/timer.hpp>
-#include <libdevcore/Assertions.h>
-#include <libdevcore/TrieHash.h>
-#include <libevm/VMFactory.h>
-#include "BlockChain.h"
 #include "Block.h"
+#include "BlockChain.h"
 #include "Defaults.h"
 #include "ExtVM.h"
 #include "TransactionQueue.h"
+#include <libdevcore/Assertions.h>
+#include <libdevcore/TrieHash.h>
+#include <libevm/VMFactory.h>
+#include <boost/filesystem.hpp>
+#include <boost/timer.hpp>
 
 using namespace std;
 using namespace dev;
 using namespace dev::eth;
 namespace fs = boost::filesystem;
 
-const char* StateSafeExceptions::name() { return EthViolet "⚙" EthBlue " ℹ"; }
-const char* StateDetail::name() { return EthViolet "⚙" EthWhite " ◌"; }
-const char* StateTrace::name() { return EthViolet "⚙" EthGray " ◎"; }
-const char* StateChat::name() { return EthViolet "⚙" EthWhite " ◌"; }
+const char* StateSafeExceptions::name()
+{
+    return EthViolet "⚙" EthBlue " ℹ";
+}
+const char* StateDetail::name()
+{
+    return EthViolet "⚙" EthWhite " ◌";
+}
+const char* StateTrace::name()
+{
+    return EthViolet "⚙" EthGray " ◎";
+}
+const char* StateChat::name()
+{
+    return EthViolet "⚙" EthWhite " ◌";
+}
 
 namespace
 {
-
 /// @returns true when normally halted; false when exceptionally halted.
-bool executeTransaction(Executive& _e, Transaction const& _t, OnOpFunc const& _onOp)
+bool executeTransaction(Executive& _e,
+                        Transaction const& _t,
```

Comment:
I don't like this when each parameter is on the separate line when all three could fit into one line on any screen.
Other opinions?

Reply:
Assuming we want to leave the column limit of 80, the options are:

```cpp
/// @returns true when normally halted; false when exceptionally halted.
bool executeTransaction(Executive& _e,
                        Transaction const& _t,
                        OnOpFunc const& _onOp)
{
    _e.initialize(_t);

    if (!_e.execute())
        _e.go(_onOp);
    return _e.finalize();
}
```

```cpp
/// @returns true when normally halted; false when exceptionally halted.
bool executeTransaction(Executive& _e, Transaction const& _t,
                        OnOpFunc const& _onOp)
{
    _e.initialize(_t);

    if (!_e.execute())
        _e.go(_onOp);
    return _e.finalize();
}
```

```cpp
/// @returns true when normally halted; false when exceptionally halted.
bool executeTransaction(Executive& _e,
    Transaction const& _t,
    OnOpFunc const& _onOp)
{
    _e.initialize(_t);

    if (!_e.execute())
        _e.go(_onOp);
    return _e.finalize();
}
```

```cpp
/// @returns true when normally halted; false when exceptionally halted.
bool executeTransaction(
    Executive& _e, Transaction const& _t, OnOpFunc const& _onOp)
{
    _e.initialize(_t);

    if (!_e.execute())
        _e.go(_onOp);
    return _e.finalize();
}
```

This seems to be a combination of 3 options, but somehow I cannot get all of them. E.g I cannot get
```cpp
/// @returns true when normally halted; false when exceptionally halted.
bool executeTransaction(Executive& _e, Transaction const& _t,
    OnOpFunc const& _onOp)
{
    _e.initialize(_t);

    if (!_e.execute())
        _e.go(_onOp);
    return _e.finalize();
}
```

Also I'm not sure we can get `)` on a new line.